meta/run.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# In this file we specify how each microservice should boot.
+#
+
+runtime: native
```

Comment:
Fixed, we're now using `runtime: node`. Not updating the `meta/package.yaml`, however, increases unikernel size by 20MB since we upload both node `4.2.6` (package.yaml) and `4.4.5` (default for runtime: node).

I think the problem that we're facing here is called "Capstan does not support package versioning yet".